executors.go

```diff
@@ -265,7 +265,8 @@ func (c *Connection) Create(model interface{}, excludeColumns ...string) error {
 							if IsZeroOfUnderlyingType(id) {
 								return c.Create(m.Value)
 							}
-							exists, errE := Q(c).Exists(i)
+
+							exists, errE := Q(c).Where("id = ?", id).Exists(i)
```

Comment:
The ID column is variable depending on the struct and should therefore not be hardcoded. `sm` has a function `whereID` which should be used instead.